Attaching to brahma_nginx_1, brahma_api2_1, brahma_api1_1, brahma_db_1
[36mapi1_1   |[0m Running migration: ./migrations/V1__initial_setup.sql
[36mapi1_1   |[0m psql:migrations/V1__initial_setup.sql:1: ERROR:  duplicate key value violates unique constraint "pg_extension_name_index"
[36mapi1_1   |[0m DETAIL:  Key (extname)=(pg_trgm) already exists.
[36mapi1_1   |[0m psql:migrations/V1__initial_setup.sql:8: ERROR:  duplicate key value violates unique constraint "pg_proc_proname_args_nsp_index"
[36mapi1_1   |[0m DETAIL:  Key (proname, proargtypes, pronamespace)=(array_to_string_immutable, 1009 25 25, 2200) already exists.
[36mapi1_1   |[0m psql:migrations/V1__initial_setup.sql:21: ERROR:  duplicate key value violates unique constraint "pg_type_typname_nsp_index"
[36mapi1_1   |[0m DETAIL:  Key (typname, typnamespace)=(people, 2200) already exists.
[36mapi1_1   |[0m psql:migrations/V1__initial_setup.sql:24: ERROR:  duplicate key value violates unique constraint "pg_class_relname_nsp_index"
[36mapi1_1   |[0m DETAIL:  Key (relname, relnamespace)=(people_id_idx, 2200) already exists.
[36mapi1_1   |[0m psql:migrations/V1__initial_setup.sql:28: ERROR:  duplicate key value violates unique constraint "pg_class_relname_nsp_index"
[36mapi1_1   |[0m DETAIL:  Key (relname, relnamespace)=(people_for_search_gin_trgm_idx, 2200) already exists.
[36mapi1_1   |[0m Migrations completed successfully.
[36mapi1_1   |[0m Listening on 80
[33mapi2_1   |[0m Running migration: ./migrations/V1__initial_setup.sql
[33mapi2_1   |[0m Migrations completed successfully.
[33mapi2_1   |[0m Listening on 80
[32mdb_1     |[0m The files belonging to this database system will be owned by user "postgres".
[32mdb_1     |[0m This user must also own the server process.
[32mdb_1     |[0m 
[32mdb_1     |[0m The database cluster will be initialized with locale "en_US.utf8".
[32mdb_1     |[0m The default database encoding has accordingly been set to "UTF8".
[32mdb_1     |[0m The default text search configuration will be set to "english".
[32mdb_1     |[0m 
[32mdb_1     |[0m Data page checksums are disabled.
[32mdb_1     |[0m 
[32mdb_1     |[0m fixing permissions on existing directory /var/lib/postgresql/data ... ok
[32mdb_1     |[0m creating subdirectories ... ok
[32mdb_1     |[0m selecting dynamic shared memory implementation ... posix
[32mdb_1     |[0m selecting default max_connections ... 100
[32mdb_1     |[0m selecting default shared_buffers ... 128MB
[32mdb_1     |[0m selecting default time zone ... Etc/UTC
[32mdb_1     |[0m creating configuration files ... ok
[32mdb_1     |[0m running bootstrap script ... ok
[32mdb_1     |[0m performing post-bootstrap initialization ... ok
[32mdb_1     |[0m syncing data to disk ... ok
[32mdb_1     |[0m 
[32mdb_1     |[0m 
[32mdb_1     |[0m Success. You can now start the database server using:
[32mdb_1     |[0m 
[32mdb_1     |[0m     pg_ctl -D /var/lib/postgresql/data -l logfile start
[32mdb_1     |[0m 
[32mdb_1     |[0m initdb: warning: enabling "trust" authentication for local connections
[32mdb_1     |[0m initdb: hint: You can change this by editing pg_hba.conf or using the option -A, or --auth-local and --auth-host, the next time you run initdb.
[32mdb_1     |[0m waiting for server to start....2023-08-23 15:03:39.035 UTC [48] LOG:  starting PostgreSQL 15.4 (Debian 15.4-1.pgdg120+1) on x86_64-pc-linux-gnu, compiled by gcc (Debian 12.2.0-14) 12.2.0, 64-bit
[32mdb_1     |[0m 2023-08-23 15:03:39.049 UTC [48] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
[32mdb_1     |[0m 2023-08-23 15:03:39.059 UTC [51] LOG:  database system was shut down at 2023-08-23 15:03:38 UTC
[32mdb_1     |[0m 2023-08-23 15:03:39.066 UTC [48] LOG:  database system is ready to accept connections
[32mdb_1     |[0m  done
[32mdb_1     |[0m server started
[32mdb_1     |[0m 
[32mdb_1     |[0m /usr/local/bin/docker-entrypoint.sh: ignoring /docker-entrypoint-initdb.d/*
[32mdb_1     |[0m 
[32mdb_1     |[0m 2023-08-23 15:03:39.220 UTC [48] LOG:  received fast shutdown request
[32mdb_1     |[0m waiting for server to shut down....2023-08-23 15:03:39.234 UTC [48] LOG:  aborting any active transactions
[32mdb_1     |[0m 2023-08-23 15:03:39.236 UTC [48] LOG:  background worker "logical replication launcher" (PID 54) exited with exit code 1
[32mdb_1     |[0m 2023-08-23 15:03:39.251 UTC [49] LOG:  shutting down
[32mdb_1     |[0m 2023-08-23 15:03:39.265 UTC [49] LOG:  checkpoint starting: shutdown immediate
[32mdb_1     |[0m 2023-08-23 15:03:39.275 UTC [49] LOG:  checkpoint complete: wrote 3 buffers (0.0%); 0 WAL file(s) added, 0 removed, 0 recycled; write=0.003 s, sync=0.002 s, total=0.024 s; sync files=2, longest=0.001 s, average=0.001 s; distance=0 kB, estimate=0 kB
[32mdb_1     |[0m 2023-08-23 15:03:39.279 UTC [48] LOG:  database system is shut down
[32mdb_1     |[0m  done
[32mdb_1     |[0m server stopped
[32mdb_1     |[0m 
[32mdb_1     |[0m PostgreSQL init process complete; ready for start up.
[32mdb_1     |[0m 
[32mdb_1     |[0m 2023-08-23 15:03:39.354 UTC [1] LOG:  starting PostgreSQL 15.4 (Debian 15.4-1.pgdg120+1) on x86_64-pc-linux-gnu, compiled by gcc (Debian 12.2.0-14) 12.2.0, 64-bit
[32mdb_1     |[0m 2023-08-23 15:03:39.354 UTC [1] LOG:  listening on IPv4 address "0.0.0.0", port 5432
[32mdb_1     |[0m 2023-08-23 15:03:39.354 UTC [1] LOG:  listening on IPv6 address "::", port 5432
[32mdb_1     |[0m 2023-08-23 15:03:39.358 UTC [1] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
[32mdb_1     |[0m 2023-08-23 15:03:39.365 UTC [62] LOG:  database system was shut down at 2023-08-23 15:03:39 UTC
[32mdb_1     |[0m 2023-08-23 15:03:39.372 UTC [1] LOG:  database system is ready to accept connections
[32mdb_1     |[0m 2023-08-23 15:03:43.901 UTC [75] ERROR:  duplicate key value violates unique constraint "pg_extension_name_index"
[32mdb_1     |[0m 2023-08-23 15:03:43.901 UTC [75] DETAIL:  Key (extname)=(pg_trgm) already exists.
[32mdb_1     |[0m 2023-08-23 15:03:43.901 UTC [75] STATEMENT:  create extension if not exists pg_trgm;
[32mdb_1     |[0m 2023-08-23 15:03:43.905 UTC [75] ERROR:  duplicate key value violates unique constraint "pg_proc_proname_args_nsp_index"
[32mdb_1     |[0m 2023-08-23 15:03:43.905 UTC [75] DETAIL:  Key (proname, proargtypes, pronamespace)=(array_to_string_immutable, 1009 25 25, 2200) already exists.
[32mdb_1     |[0m 2023-08-23 15:03:43.905 UTC [75] STATEMENT:  create or replace function array_to_string_immutable (
[32mdb_1     |[0m 	    arg text[],
[32mdb_1     |[0m 	    separator text,
[32mdb_1     |[0m 	    null_string text default null)
[32mdb_1     |[0m 	returns text immutable parallel safe language sql as $$
[32mdb_1     |[0m 	select array_to_string(arg,separator,null_string) $$;
[32mdb_1     |[0m 2023-08-23 15:03:43.916 UTC [75] ERROR:  duplicate key value violates unique constraint "pg_type_typname_nsp_index"
[32mdb_1     |[0m 2023-08-23 15:03:43.916 UTC [75] DETAIL:  Key (typname, typnamespace)=(people, 2200) already exists.
[32mdb_1     |[0m 2023-08-23 15:03:43.916 UTC [75] STATEMENT:  create table if not exists people (
[32mdb_1     |[0m 	    id         uuid unique ,
[32mdb_1     |[0m 	    apelido    varchar(32) unique,
[32mdb_1     |[0m 	    nome       varchar(100),
[32mdb_1     |[0m 	    nascimento date,
[32mdb_1     |[0m 	    stack      text[] null,
[32mdb_1     |[0m 	    for_search text generated always as (
[32mdb_1     |[0m 	        lower(apelido) || ' ' ||
[32mdb_1     |[0m 	        lower(nome)    || ' ' ||
[32mdb_1     |[0m 	        coalesce(lower(array_to_string_immutable(stack, ' '::text)), ''::text)
[32mdb_1     |[0m 	    ) stored
[32mdb_1     |[0m 	);
[32mdb_1     |[0m 2023-08-23 15:03:43.920 UTC [75] ERROR:  duplicate key value violates unique constraint "pg_class_relname_nsp_index"
[32mdb_1     |[0m 2023-08-23 15:03:43.920 UTC [75] DETAIL:  Key (relname, relnamespace)=(people_id_idx, 2200) already exists.
[32mdb_1     |[0m 2023-08-23 15:03:43.920 UTC [75] STATEMENT:  create index if not exists people_id_idx
[32mdb_1     |[0m 	on people (id);
[32mdb_1     |[0m 2023-08-23 15:03:43.922 UTC [75] ERROR:  duplicate key value violates unique constraint "pg_class_relname_nsp_index"
[32mdb_1     |[0m 2023-08-23 15:03:43.922 UTC [75] DETAIL:  Key (relname, relnamespace)=(people_for_search_gin_trgm_idx, 2200) already exists.
[32mdb_1     |[0m 2023-08-23 15:03:43.922 UTC [75] STATEMENT:  create index if not exists people_for_search_gin_trgm_idx
[32mdb_1     |[0m 	on people
[32mdb_1     |[0m 	using gin (for_search gin_trgm_ops);
[35mnginx_1  |[0m /docker-entrypoint.sh: /docker-entrypoint.d/ is not empty, will attempt to perform configuration
[35mnginx_1  |[0m /docker-entrypoint.sh: Looking for shell scripts in /docker-entrypoint.d/
[35mnginx_1  |[0m /docker-entrypoint.sh: Launching /docker-entrypoint.d/10-listen-on-ipv6-by-default.sh
[35mnginx_1  |[0m 10-listen-on-ipv6-by-default.sh: info: Getting the checksum of /etc/nginx/conf.d/default.conf
[35mnginx_1  |[0m 10-listen-on-ipv6-by-default.sh: info: Enabled listen on IPv6 in /etc/nginx/conf.d/default.conf
[35mnginx_1  |[0m /docker-entrypoint.sh: Sourcing /docker-entrypoint.d/15-local-resolvers.envsh
[35mnginx_1  |[0m /docker-entrypoint.sh: Launching /docker-entrypoint.d/20-envsubst-on-templates.sh
[35mnginx_1  |[0m /docker-entrypoint.sh: Launching /docker-entrypoint.d/30-tune-worker-processes.sh
[35mnginx_1  |[0m /docker-entrypoint.sh: Configuration complete; ready for start up
